logicrunner/builtin/contract/member/member.go

```diff
@@ -197,9 +203,35 @@ func (m *Member) registerNodeCall(params map[string]interface{}) (interface{}, e
 
 	return m.registerNode(publicKey, role)
 }
-func (m *Member) addBurnAddressesCall(params map[string]interface{}) (interface{}, error) {
 
-	burnAddressesI, ok := params["burnAddresses"].([]interface{})
+func (m *Member) migrationAdminCall(params map[string]interface{}, nameMethod string) (interface{}, error) {
+	migrationAdminContract := migrationadmin.GetObject(foundation.GetMigrationAdmin())
+
+	switch nameMethod {
```

Comment:
where are files with tests for getInfo, activateDaemon and deactivateDaemon

Reply:
in next task https://insolar.atlassian.net/browse/INS-3027. for this you need migration.getInfo. from task https://insolar.atlassian.net/browse/INS-3027